CMakeLists.txt

```diff
@@ -15,15 +15,20 @@ pkg_check_modules(DLT REQUIRED automotive-dlt-c++)
 
 add_subdirectory( deps/audio/common )
 
+#SET(GPTP_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
If this is unnecessary, just remove the code, not comment out. Oh, and no need to send another pr for this, simply remove and merge =D